services/UserService.js

```diff
@@ -49,6 +50,17 @@ class UserService {
     const hashedPassword = await CryptographyService.hashPassword(password);
     await UserModel.updatePassword(id, hashedPassword);
   }
+
+  async updateAvatar(id, image) {
+    const user = await UserModel.findById(id);
+    if (!user) {
+      throw new Error('User not found!');
+    }
+    const imageData = await AWS.upload(image, id);
```

Comment:
optional: use more meaningful name, because aws has a lot of services + its same with aws module that we extract